ci-operator/step-registry/openshift-extended/test/openshift-extended-test-commands.sh

```
@@ -271,6 +271,42 @@ if [[ $IS_ACTIVE_CLUSTER_OPENSHIFT != "false" ]]; then
     oc get clusterversion version -o yaml || true
 fi
 
+# Debug wait with timeout (default 3 hours) - placed AFTER cluster is ready
+DEBUG_WAIT_TIMEOUT="${DEBUG_WAIT_TIMEOUT:-10800}"
+echo "=========================================="
+echo "DEBUG WAIT ACTIVE - Cluster ready for debugging"
+echo "=========================================="
+echo "Cluster is fully provisioned and ready!"
+echo "Timeout: ${DEBUG_WAIT_TIMEOUT} seconds ($(($DEBUG_WAIT_TIMEOUT / 3600)) hours)"
```

Shellcheck (0.11.0)

[style] $/${} is unnecessary on arithmetic variables.

(SC2004)